addons/web-support/api/src/main/java/org/jboss/windup/web/addons/websupport/rest/graph/applicationDetails/ApplicationMessageReducedDTO.java

```diff
@@ -0,0 +1,50 @@
+package org.jboss.windup.web.addons.websupport.rest.graph.applicationDetails;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import javax.enterprise.inject.Vetoed;
+
+/**
+ * Contains messages that apply at the application level.
+ *
+ * @author <a href="mailto:jesse.sightler@gmail.com">Jesse Sightler</a>
+ */
+@Vetoed
+@JsonIgnoreProperties({"handler", "delegate"})
```

Comment:
Hmm is this one there because of dependency injection adds some special properties?

I faced similar problem and didn't know exactly how to solve it, so I created map instead. But it would be nice to have some generic solution and use it everywhere we face this problem.